### ISSUE TITLE
start pgcli error after installation

### ISSUE THREAD

**Issue by stardiviner** [dbcli/pgcli#309]
I got this error when I start it with `pgcli` after installation:

```
 ⇨   pgcli                                                                                                          Hack
Traceback (most recent call last):
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.4/site-packages/pkg_resources/__init__.py", line 612, in _build_master
    ws.require(__requires__)
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.4/site-packages/pkg_resources/__init__.py", line 918, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.4/site-packages/pkg_resources/__init__.py", line 810, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (prompt-toolkit 0.45 (/home/stardiviner/.virtualenvs/python3/lib/python3.4/site-packages), Requirement.parse('prompt-toolkit==0.42'), {'pgcli'})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/stardiviner/.virtualenvs/python3/bin/pgcli", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.4/site-packages/pkg_resources/__init__.py", line 3018, in <module>
    working_set = WorkingSet._build_master()
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.4/site-packages/pkg_resources/__init__.py", line 614, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.4/site-packages/pkg_resources/__init__.py", line 627, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.4/site-packages/pkg_resources/__init__.py", line 805, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: prompt-toolkit==0.42
```

This looks like version conflict or something else.
I tried to uninstall `prompt-toolkit`, and install it again, still error.

**Comment by stardiviner** [comment]
Solved:
install dependency versions:

- =$ pip install prompt-toolkit\equal{}\equal{}0.42=
- =$ pip install prompt-toolkit\equal{}\equal{}0.38=

**Comment by stardiviner** [comment]
Why pgcli does not auto installed required dependency version `prompt-toolkit`?

**Comment by stardiviner** [comment]
This problem is not solved yet.
I found `pgcli` require different version `prompt-toolkit` with `mycli`.
pgcli need prompt-toolkit==0.42
mycli need prompt-toolkit==0.38
This is weird.

**Comment by amjith** [comment]
Yes. This is a known issue. I wil be updating mycli to match pgcli and releasing a new version soon. 

Until then I'd recommend using virtualenvs. http://docs.python-guide.org/en/latest/dev/virtualenvs/#virtual-environments

Sorry about the inconvenience.

**Comment by amjith** [comment]
This is now fixed in the latest release of pgcli (0.19.1). Prompt Toolkit is upgraded to 0.45.

**Comment by stardiviner** [comment]
@amjith Really fast work. thanks very much
